### PR TITLE
Add QA jobs for COS milestone 63

### PR DESCRIPTION
**Pull request by adityakali**
/cc @kubernetes/goog-image @wonderfly

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -2025,6 +2025,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-m63": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=gci/gci-63",
+      "--gcp-project-type=gci-qa-project",
```
Comment (wonderfly) [inline jobs/config.json]: I didn't check but please make sure gci-63 this exists before checking in.

**Comment by krzyzacy** [comment]
add the new testgrid entries to https://github.com/adityakali/test-infra/blob/dcf2c38922d39ba280f6686243ff962ff63ea7f1/testgrid/config/config.yaml#L2016

**Comment by wonderfly** [comment]
New jobs will also need to be added to dashboards, e.g., `test_group_name: ci-kubernetes-e2e-gce-gci-qa-m62`

File: jobs/config.json (hunk 1 above)
Comment (adityakali) [reply jobs/config.json]: It does now. Thanks for reminder.

**Comment by adityakali** [comment]
Added entries to `google-gci-dev` tab on testgrid.
PTAL.

**Pull request merged**